recipes/cfunits/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "cfunits" %}
+{% set version = "1.5.1" %}
+{% set sha256 = "3735c1c9fea8e3b5b37094843b590a2dc856be3bc7df44d2476ad51902b04692" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
+  script: python setup.py install --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
It doesn't look like `setuptools` is used in `setup.py`, so it should be removed.